apps/aecuckoo/Makefile

```diff
@@ -1,24 +1,42 @@
 EXECUTABLES = \
-	priv/bin/mean16 \
-	priv/bin/lean16 \
-	priv/bin/verify16 \
-	priv/bin/mean28 \
+	priv/bin/mean30s-generic \
+	priv/bin/mean28s-generic \
+	priv/bin/mean16s-generic \
+	priv/bin/mean30s-avx2 \
+	priv/bin/mean28s-avx2 \
+	priv/bin/mean16s-avx2 \
+	priv/bin/lean30 \
 	priv/bin/lean28 \
+	priv/bin/lean16 \
+	priv/bin/verify30 \
```

Comment:
Except for the "smoke test" is there a point in including the verifiers? We don't use them anymore I realise...

Reply:
I think it is useful from a user or operator perspective to have a consistent set of upstream binaries so command-line checks can be performed. I can remove them if that is the agreement though...